Khan/wonder-blocks
# WB-322: buttons on Safari have extra 2px left/right margin

This PRs removes extra padding around Buttons that use <button> on Safari.  It also updates `yarn start` to run `styleguidist` and `webpack -w` concurrently.

Test Plan:
- load http://localhost:6060 in Safari
- inspect the buttons in the inspector, see that they don't have any left padding (the right padding is added by the example, not Safari's built-in styles)

Before:
<img width="609" alt="screen shot 2018-07-20 at 6 45 44 pm" src="https://user-images.githubusercontent.com/1044413/43028692-4b5730e4-8c4f-11e8-8d84-b4b5f36625c6.png">

After:
<img width="593" alt="screen shot 2018-07-20 at 6 45 07 pm" src="https://user-images.githubusercontent.com/1044413/43028698-5084093e-8c4f-11e8-943b-47dbb109df85.png">
